var/spack/repos/builtin/packages/openloops/get-process-list.py

```diff
@@ -8,7 +8,7 @@ def error(self, message):
         pass
 
     def __init__(self):
-        super().__init__()
+        super(HTMLParser).__init__()

```

Comment:
```suggestion
        super(HTMLParser, self).__init__()
```
I think this is the correct syntax?

Reply:
Yes, sorry. Was a little too fast there. :-)